src/Diagnostics/FxCop/CSharp/Microsoft.CodeAnalysis.CSharp.FxCopAnalyzers.props

```diff
@@ -8,7 +8,6 @@
   <PropertyGroup>
     <CodeAnalysisRuleSetOverrides>$(CodeAnalysisRuleSetOverrides);
       -Microsoft.Design#CA1003;
-      -Microsoft.Design#CA1019;
```

Comment:
We should probably write an analyzer to verify that the props file always stays up-to-date:

1. It has no extra entries for IDs that are not part of any analyzer in owing project.
2. Every newly added CA analyzer has an entry in here.

Logically, an equivalent of our DeclarePublicAPI analyzer. This would be helpful as we port more and more rules to FXCop analyzer projects.